scripts/deploy-frontend.sh

```diff
@@ -19,29 +19,41 @@ update_frontend()
     cd tdrs-frontend || exit
 
     if [ "$CF_SPACE" = "tanf-prod" ]; then
-        echo "REACT_APP_BACKEND_URL=https://api-tanfdata.acf.hhs.gov/v1" >> .env.production
-        echo "REACT_APP_BACKEND_HOST=https://api-tanfdata.acf.hhs.gov" >> .env.production
+        echo "REACT_APP_BACKEND_URL=https://tanfdata.acf.hhs.gov/v1" >> .env.production
```

Comment:
@raftmsohani is this correct? prod backend URL begins with `https://api-tanfdata`

Reply:
Yes, after merge all urls are served by Nginx which has frontend url